client/src/components/scores/score.js

```diff
@@ -3,51 +3,57 @@
 export default {
 	template: `
 	<div class="card-section" ng-class="{ loading: loading }">

```

Comment:
In order to give the right padding, when every field is a card-section, This div should not be a card section

Reply:
Also, `loading` in general doesn't refer to anything, because it's not based on the controller `score`. It should be `score.loading`

Reply:
Who sets this `loading` to `true`?

Reply:
As for the loading, I have no idea. It is like this in dev/standard. I did not touch it.